frontend/src/components/HomePage/HomePage.jsx

```diff
@@ -1,17 +1,71 @@
 import styles from './HomePage.module.css';
 import { Link, useLocation } from 'react-router-dom';
 import React, { useState } from 'react';
+import SelectTime from '../Select/SelectTime';
+
+const buttonStyles = {
+  padding: '1.2rem',
+  margin: '1rem',
+  border: '1px solid black',
+  background: 'transparent',
+  borderRadius: '1rem',
+  display: 'flex',
+  flexDirection: 'column'
+};
+
+const containerStyles = {
+  display: 'flex'
+};
 
 export default function HomePage() {
   const [data] = useState({});
   const { state } = useLocation();
+  const [status, setStatus] = useState('available');
+  const [setTime] = useState(null);
+
   console.log(' from homepage.js state ', state);
   console.log(' from homepage.js data', data);
   return (
     <>
       <div className={styles.center}>
-        <span className={styles.largeIcon}>account_circle</span>
-
+        <div className={styles.subTitle}>
+          <h4>My status</h4> <SelectTime setTime={setTime} />
```

Comment:
@ValentinaBacherer I'm wondering - it's not clear to the user what this minute component is when you look at the app. I think for MVP it's ok to remove. What do you think?